Gemfile

```diff
@@ -83,6 +83,8 @@ gem 'nio4r', '= 2.5.8'
 gem 'racc', '= 1.6.2'
 gem 'websocket-driver', '= 0.7.5'
 
+gem 'simplecov-json', :require => false, :group => :test
```

Comment:
if this project isn't setup for codecoverage, remove test reporter from CI, and tackle this as part of another PR